libs/rollup.config.ts

```diff
@@ -0,0 +1,278 @@
+import commonjs from '@rollup/plugin-commonjs'
+import babel from '@rollup/plugin-babel'
+import json from '@rollup/plugin-json'
+import resolve from '@rollup/plugin-node-resolve'
+import typescript from '@rollup/plugin-typescript'
+import { terser } from 'rollup-plugin-terser'
+import nodePolyfills from 'rollup-plugin-polyfill-node'
+import alias from '@rollup/plugin-alias'
+import ignore from 'rollup-plugin-ignore'
+
+import pkg from './package.json'
+
+const extensions = ['.js', '.ts']
+
+const external = [
+  ...Object.keys(pkg.dependencies),
+  ...Object.keys(pkg.devDependencies),
+  'ethers/lib/utils',
+  'ethers/lib/index',
+  'hashids/cjs'
+]
+
+const pluginTypescript = typescript({ tsconfig: './tsconfig.json' })
+
+/**
+ * For the browser bundle, these need to be internal because they either:
+ * - contain deps that need to be polyfilled via `nodePolyfills`
+ * - are ignored via `ignore`
+ */
+const browserInternal = [
+  'eth-sig-util',
+  'ethereumjs-tx',
+  'ethereumjs-util',
+  'ethereumjs-wallet',
+  'graceful-fs',
+  'node-localstorage',
+  'abi-decoder',
+  'web3',
+  'xmlhttprequest'
+]
+
+/**
+ * ES-only dependencies need inlining when outputting a Common JS bundle,
+ * as requiring ES modules from Common JS isn't supported.
+ * Alternatively, these modules could be imported using dynamic imports,
+ * but that would have other side effects and affect each bundle output
+ * vs only affecting Common JS outputs, and requires Rollup 3.0.
+ *
+ * TODO: Make a test to ensure we don't add external ES-only modules to Common JS output
+ *
+ * See:
+ * - https://nodejs.org/api/esm.html#interoperability-with-commonjs
+ * - https://github.com/rollup/plugins/issues/481#issuecomment-661622792
+ * - https://github.com/rollup/rollup/pull/4647 (3.0 supports keeping dynamic imports)
+ */
+const commonJsInternal = ['micro-aes-gcm']
+
+export const outputConfigs = {
+  /**
+   * SDK (and Libs) Node Package (Common JS)
+   * Used by the Audius Content Node Service and Identity Service
+   * - Includes libs
+   * - Makes external ES modules internal to prevent issues w/ using require()
+   */
+  sdkConfigCjs: {
+    input: 'src/index.ts',
+    output: [{ file: pkg.main, format: 'cjs', sourcemap: true }],
+    plugins: [
+      resolve({ extensions, preferBuiltins: true }),
+      commonjs({ extensions }),
+      babel({ babelHelpers: 'bundled', extensions }),
+      json(),
+      pluginTypescript
+    ],
+    external: external.filter((id) => !commonJsInternal.includes(id))
+  },
+
+  /**
+   * SDK (and Libs) Node Package (ES Module)
+   * Potentially used by third parties?
```

Comment:
It is used by third parties, we added this bundle due to a request for ESM but I don't remember which third party asked for it